classes/mender-uboot.bbclass

```diff
@@ -7,6 +7,3 @@ IMAGE_BOOT_ENV_FILE ?= "uboot.env"
 IMAGE_BOOT_FILES_append = " ${IMAGE_BOOT_ENV_FILE}"
 
 EXTRA_IMAGEDEPENDS += "u-boot"
-
-DEPENDS_mender_append = " u-boot u-boot-fw-utils"
-RDEPENDS_mender_append = " u-boot u-boot-fw-utils"
```

Comment:
This chunk is actually removing hard coded dependency of mender on uboot, so what's the issue here?

Besides, the only real dependency we have now is on uboot-fw-utils and that is added to RDEPENDS.